app/controllers/api/customers_controller.rb

```diff
@@ -7,38 +7,34 @@ def index
   def update
     @customer = Customer.find_by(id: params[:id])
 
-    @customer.update(first_name: params[:address][:first_name],
-                      last_name: params[:address][:last_name],
-                      address: params[:address][:address],
-                      Address2: params[:address][:Address2],
-                      zip_code: params[:address][:zip_code],
-                      city: params[:address][:city],
-                      state: params[:address][:state])
-    stripe_customer = Stripe::Customer.retrieve(@customer.stripe_customer_id)
-
-
-    carted_products = CartedProduct.my_carted(guest_or_customer_id)
-
-    items = carted_products.map{ |o| {type: 'sku', parent: o.sku, quantity: o.quantity } }
+    full_name = "#{params[:address][:first_name]} #{params[:address][:last_name]}"
```

Comment:
@JHughesSUTA Take a look at the [PR](https://github.com/the-difference-engine/back-yards-coffee-app/pull/177) I submitted...I moved the `full_name` to the `Customer` class.